cmake/BuildInfo.cmake

```diff
@@ -0,0 +1,14 @@
+function(mrtrix_print_build_instructions)
+    set(message_list "\n")
+    list(APPEND message_list "MRtrix3 has been successfully configured for compilation.")
+    list(APPEND message_list "To build MRtrix3, run 'cmake --build .' from the build directory.")
```

Comment:
Personally I tend to build from the source root, and it would be expected that the source root would be the current working directory when running the configuration step. So would it be better to have the example build invocation refer to the relative path to the build directory by name (eg. `"To build MRtrix3, run: 'cmake --build build/'."`)?

Reply:
That makes sense to me. I've now changed the instructions to call 'cmake --build build_directory' instead